lms/static/js/spec/staff_debug_actions_spec.js

```diff
@@ -41,9 +49,45 @@ define(['backbone', 'jquery', 'js/staff_debug_actions'],
                 it('gets the placeholder name if the id has escapable characters', function() {
                     $('body').append(escapableFixture);
                     expect(StaffDebug.get_user('test.*+?^:${}()|][loc')).toBe('userman');
-                    $('#' + escapableFixture_id).remove();
+                    $("input[id^='sd_fu_']").remove();
                 });
             });
+            describe('do_idash_action success', function () {
```

Comment:
I've left the descriptions in the 'describe' part as is, because they match the rest of the file, but I've added a more descriptive description in the 'it' segment.